### PR TITLE
Proposal - more concrete explanation of the fail for __set_state

### PR DESCRIPTION
**Pull request by AlexandrErohin**
It is proposal to explain more concrete the type of the fail after re-importing objects from var_export.
That it is not only would not be re-imported - but also causes fatal error

### CODE REVIEW

File: language/oop5/magic.xml [hunk 1]
```diff
@@ -386,8 +386,8 @@ object(A)#2 (2) {
     <simpara>
      When exporting an object, <function>var_export</function> does not check
      whether <link linkend="object.set-state">__set_state()</link> is
-     implemented by the object's class, so re-importing such objects will fail,
-     if __set_state() is not implemented. Particularly, this affects some
+     implemented by the object's class, so re-importing such objects will result in a Fatal Error,
```
Comment (afilina) [inline language/oop5/magic.xml:389]: Is this true for all PHP versions, or does this have a different result based on the version?

**Comment by AlexandrErohin** [comment]
@afilina 
Yes - it results with Fatal error since version 5.1.0
https://3v4l.org/C6SrZ

**Comment by cmb69** [comment]
Actually, that is an `Error` exception as of PHP 7.0.0 (older versions are no longer relevant for the manual).

File: language/oop5/magic.xml (hunk 1 above)
Comment (afilina) [inline language/oop5/magic.xml:389]:
```suggestion
     implemented by the object's class, so re-importing such objects will result in an <classname>Error</classname> exception,
```

**Review by afilina — CHANGES_REQUESTED** [review]

File: language/oop5/magic.xml (hunk 1 above)
Comment (AlexandrErohin) [reply language/oop5/magic.xml]: @afilina Thank you!

**Review by afilina — APPROVED** [review]

File: language/oop5/magic.xml [hunk 2]
```diff
@@ -386,8 +386,8 @@ object(A)#2 (2) {
     <simpara>
      When exporting an object, <function>var_export</function> does not check
      whether <link linkend="object.set-state">__set_state()</link> is
-     implemented by the object's class, so re-importing such objects will fail,
-     if __set_state() is not implemented. Particularly, this affects some
+     implemented by the object's class, so re-importing such objects will result in an <classname>Error</classname> exception,
+     as __set_state() is undefined. Particularly, this affects some
```
Comment (cmb69) [inline language/oop5/magic.xml:390]:
```suggestion
     implemented by the object's class, so re-importing objects will result in an <classname>Error</classname> exception,
     if __set_state() is not implemented. Particularly, this affects some
```

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**